lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -527,6 +546,11 @@ function setupEventListeners(transport) {
         }
       case 'connecting':
         switch (message.type) {
+          case 'iced':
+            transport._options.onIced(message.ice_servers).then(() => {
+              transport._sendConnectOrSyncOrDisconnectMessage();
+            });
```

Comment:
we should .catch` case here and disconnect. This will happen when user cancels the promise.

Reply:
@makarandp0 Disconnect is handled by the rejection of the CancelablePromise.